.github/workflows/actions/get-core-dependencies/action.yml

```diff
@@ -0,0 +1,24 @@
+name: 'Get Core Dependencies'
+description: 'initializes core dependencies after repo has been pulled'
+runs:
+  using: composite
+  steps:
+    - name: Use Node 16
+      uses: actions/setup-node@56337c425554a6be30cdef71bf441f15be286854 # v3.1.1
+      with:
+        node-version: 16
+        cache: 'npm'
```

Comment:
I want to check my understanding here - does this mean that the dependency install is always going to happen on node 16? I wonder if we could run into an issue where a `postInstall` script runs correctly on node 16 but would error on node 14 - possibly this isn't really something to worry about though

Reply:
>  does this mean that the dependency install is always going to happen on node 16?

Yeah, that's correct. At the moment we do the install/build/publish all from Node 16 (manually, although we have plans to change that), so I considered the risk of a `postInstall` script running differently in a separate version of Node to be minimal.

Reply:
that makes sense 👍